cluster-state-service/handler/reconcile/loader/instanceloader.go

```diff
@@ -124,6 +124,7 @@ func (loader instanceLoader) loadLocalClusterStateFromStore() (clusterARNsToInst
 }
 
 // getContainerInstancesFromECS gets a list of container instances from ECS for the specified cluster.
+// The ECS ListContainerInstances method returns active and draining container instances. It does not return inactive container instances.
```

Comment:
Are both returned by default? Or just active? If it's the latter, can we update the comment to clarify it?